snap/snapcraft.yaml

```diff
@@ -0,0 +1,20 @@
+
+name: mudlet # you probably want to 'snapcraft register <name>'
+version: '3.1.0' # just for humans, typically '1.2+git' or '1.3.2'
+summary: A cross-platform, open source, and super fast MUD client with scripting in Lua
+description: |
+  Mudlet has an intuitive user interface, a specially designed scripting framework, and a very fast text display. Add to that cross-platform capability, an open-source development model, and you have a very likable MUD client.
+
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: devmode # use 'strict' once you have the right plugs and slots
+
+apps:
+  mudlet:
+    command: desktop-launch mudlet
+    plugs: [home, x11, opengl, pulseaudio]
+
+parts:
+  mudlet:
+    plugin: cmake
```

Comment:
Isn't there a `qmake` plugin as well? Do we really want to use `cmake` for snaps?